packages/flutterfire_ui/lib/src/firestore/table_builder.dart

```diff
@@ -58,12 +70,24 @@ class FirestoreDataTable extends StatefulWidget {
     this.dragStartBehavior = DragStartBehavior.start,
     this.arrowHeadColor,
     this.checkboxHorizontalMargin,
+    this.cellBuilder,
+    this.canEditCell = true,
+    this.onEditItem,
   })  : assert(
           columnLabels is LinkedHashMap,
           'only LinkedHashMap are supported as header',
         ), // using an assert instead of a type because `<A, B>{}` types as `Map` but is an instance of `LinkedHashMap`
         super(key: key);
 
+  /// When specified, the builder will be use to disply your own widget for the cel
+  final CellBuilder? cellBuilder;
+
+  /// When set to false this allow row selection, defaults is set to true
+  final bool canEditCell;
+
+  /// When specified, this will be called when selecting a cel
```

Comment:
```suggestion
  /// When specified, this will be called when selecting a cell
```

Reply:
Same remark here. I don't think the word `selecting` makes sense here when considering that we're talking about edits

Reply:
I had in mind desktop, since I'm using it on web :-)

What about ->

When specified, this will be called when tapping/selecting a cell